cylc/uiserver/resolvers.py

```diff
@@ -529,31 +539,42 @@ async def mutator(
             'graphql', w_ids, graphql_args, req_meta=req_meta
         )
 
-    async def service(
-        self,
-        info: 'ResolveInfo',
-        command: str,
-        workflows: Iterable['Tokens'],
-        kwargs: Dict[str, Any]
-    ) -> List[Union[bool, str]]:
+    async def get_service(self, command, kwargs, workflows):
+        method = getattr(Services, command)
         if command == 'clean':
-            return await Services.clean(
+            mutation_response = partial(
```

Comment:
@oliver-sanders I don't think we need `functools.partial` for this kind of pattern?

Reply:
Just saw the comment above

> refactored multiple if else into dictionary (because flake8 told me to)

In this case I don't see a problem with multiple `else if`; I'm happy with a `# noqa: <err code>` comment being added to the offending line

Reply:
More details here:
https://github.com/cylc/cylc-uiserver/pull/493#discussion_r1322843810
But yea - I think I probably shouldnt have touched what was there in the first place